gui/src/app/pages/HomePage/DataGenerationWindow/DataPyFileEditor.tsx

```diff
@@ -87,6 +87,21 @@ const DataPyFileEditor: FunctionComponent<Props> = ({
     [fileContent, editedFileContent, handleRun, status, handleHelp],
   );
 
+  const contentOnEmpty = useMemo(() => {
+    const spanElement = document.createElement("span");
+    const t1 = document.createTextNode(
+      "Define a dictionary called data to update the data.json. ",
+    );
+    const a1 = document.createElement("a");
+    a1.onclick = () => {
+      setEditedFileContent(dataPyTemplate);
+    };
+    a1.textContent = "Click here to generate an example";
+    spanElement.appendChild(t1);
+    spanElement.appendChild(a1);
+    return spanElement;
+  }, [setEditedFileContent]);
```

Comment:
Is it possible to do this as a React component? No worries if not, it just seems like it'd be more straightforward than having to assemble it manually.

Reply:
Would be nice, but I don't think it's possible.

Reply:
I just looked into a few options, all the supported ways of turning a JSX element into a actual DOM element are in `react-dom/server` and therefore don't persist the `onClick` in a meaningful way. Too bad